scripts/utils/codemod/create-processor.js

```diff
@@ -9,7 +9,6 @@ const createProcessor = ({ codemods }) => {
   const processor = unified()
     .use(stringify, {
       bullet: '*',
-      commonmark: true,
```

Comment:
sowe aren't using commonmark? what did removing this do out of curiosity?

Reply:
It was wreaking havoc on some of the output and causing the build to fail, mostly because it added a comment separator in some cases, which are not recognized by MDX. For now I think this is safe to turn off.

Reply:
okay! thank you